core/src/main/java/io/grpc/internal/DnsNameResolver.java

```diff
@@ -278,10 +298,15 @@ static boolean jndiAvailable() {
   static final class ResolutionResults {
     final List<InetAddress> addresses;
     final List<String> txtRecords;
+    final List<EquivalentAddressGroup> srvRecords;
 
-    ResolutionResults(List<InetAddress> addresses, List<String> txtRecords) {
+    ResolutionResults(
+        List<InetAddress> addresses,
+        List<String> txtRecords,
+        List<EquivalentAddressGroup> srvRecords) {
```

Comment:
These are actually balancer addresses. They are not intended to be generic SRV records. Please name the variables (including those in CompositeResolver) accordingly.

Reply:
Done.